dev_docs/tutorials/data/search.mdx

```diff
@@ -0,0 +1,496 @@
+---
+id: kibDevTutorialDataSearchAndSessions
+slug: /kibana-dev-docs/tutorials/data/search-and-sessions
+title: Kibana data.search Services
+summary: Kibana Search Services 
+date: 2021-02-10
+tags: ['kibana', 'onboarding', 'dev', 'tutorials', 'search', 'sessions', 'search-sessions']
+---
+
+## Search service
+
+### Low level search 
+
+Searching data stored in Elasticsearch can be done in various ways, for example using the Elasticsearch REST API or using an `Elasticsearch Client` for low level access.
+
+However, the recommended and easist way to search Elasticsearch is by using the low level search service. The service is exposed from the `data` plugin, and by using it, you not only gain access to the data you stored, but also to capabilities, such as Custom Search Strategies, Asynchronous Search, Partial Results, Search Sessions, and more.
```

Comment:
I am actually recommending the opposite here: https://github.com/elastic/kibana/pull/92743/files#diff-21850b7db15ddf8a1b1ca9dd3a3351160f41bfcb29453b386611b5aaded15d10R95

I think we should be promoting the higher-level search source as the first search utility to consider using because it will start plugin developers off on the right path of using our higher-level constructs like Index Patterns.